File: QuickStarts/CAB/VB/BankTeller/BankTellerCommon/BankTellerConstants.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Text

Namespace BankTellerCommon
	Public Class UIExtensionConstants
		Public Shared ReadOnly CUSTOMERCONTEXT As String = "CustomerContext"

		Public Shared ReadOnly FILE As String = "File"

		Public Shared ReadOnly MAINMENU As String = "MainMenu"

		Public Shared ReadOnly FILEDROPDOWN As String = "FileDropDown"

		Public Shared ReadOnly MAINSTATUS As String = "MainStatus"

		Public Shared ReadOnly QUEUE As String = "Queue"

		Public Shared ReadOnly CUSTOMER As String = "Customer"
	End Class

	''' <summary>
	''' Used for creating and handling commands.
	''' These are constants because they are used in attributes.
	''' </summary>
	Public Class CommandConstants
		Public Const ACCEPT_CUSTOMER As String = "QueueAcceptCustomer"

		Public Const EDIT_CUSTOMER As String = "EditCustomer"

		Public Const CUSTOMER_MOUSEOVER As String = "CustomerMouseOver"
	End Class

	''' <summary>
	''' Used for handling State.
	''' These are constants because they are used in attributes.
	''' </summary>
	Public Class StateConstants
		Public Const CUSTOMER As String = "Customer2"
	End Class

	Public Class WorkspacesConstants
		Public Shared ReadOnly SHELL_SIDEBAR As String = "sideBarWorkspace"
		Public Shared ReadOnly SHELL_CONTENT As String = "contentWorkspace"
	End Class
End Namespace

File: QuickStarts/CAB/CS/BankTeller/BankTellerCommon/BankTellerConstants.cs
//===============================================================================
// Microsoft patterns & practices
// Smart Client Software Factory 2010
//===============================================================================
// Copyright (c) Microsoft Corporation.  All rights reserved.
// THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY
// OF ANY KIND, EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT
// LIMITED TO THE IMPLIED WARRANTIES OF MERCHANTABILITY AND
// FITNESS FOR A PARTICULAR PURPOSE.
//===============================================================================
// The example companies, organizations, products, domain names,
// e-mail addresses, logos, people, places, and events depicted
// herein are fictitious.  No association with any real company,
// organization, product, domain name, email address, logo, person,
// places, or events is intended or should be inferred.
//===============================================================================
using System;
using System.Collections.Generic;
using System.Text;

namespace BankTellerCommon
{
	public class UIExtensionConstants
	{
		public static readonly string CUSTOMERCONTEXT = "CustomerContext";

		public static readonly string FILE = "File";

		public static readonly string MAINMENU = "MainMenu";

		public static readonly string FILEDROPDOWN = "FileDropDown";

		public static readonly string MAINSTATUS = "MainStatus";

		public static readonly string QUEUE = "Queue";

		public static readonly string CUSTOMER = "Customer";
	}

	/// <summary>
	/// Used for creating and handling commands.
	/// These are constants because they are used in attributes.
	/// </summary>
	public class CommandConstants
	{
		public const string ACCEPT_CUSTOMER = "QueueAcceptCustomer";

		public const string EDIT_CUSTOMER = "EditCustomer";

		public const string CUSTOMER_MOUSEOVER = "CustomerMouseOver";
	}

	/// <summary>
	/// Used for handling State.
	/// These are constants because they are used in attributes.
	/// </summary>
	public class StateConstants
	{
		public const string CUSTOMER = "Customer2";
	}

	public class WorkspacesConstants
	{
		public static readonly string SHELL_SIDEBAR = "sideBarWorkspace";
		public static readonly string SHELL_CONTENT = "contentWorkspace";
	}
}
